README.md

```diff
@@ -2,33 +2,46 @@
 [![npm version](https://img.shields.io/npm/v/single-spa.svg?style=flat-square)](https://www.npmjs.org/package/single-spa)
 [![Build Status](https://img.shields.io/travis/CanopyTax/single-spa.svg?style=flat-square)](https://travis-ci.org/CanopyTax/single-spa)
 
-Combine multiple SPAs into one SPA by implementing lifecycle functions. Allows you to:
-- [Trade off between frameworks](/docs/single-spa-ecosystem.md#help-for-frameworks) [without refreshing the page](/docs/child-applications.md)
+Combine multiple SPAs into one SPA by implementing lifecycle functions. This allows you to:
+- [Use multiple frameworks on the same page](/docs/single-spa-ecosystem.md#help-for-frameworks) [without refreshing the page](/docs/child-applications.md)
   (React, Angular 1, Angular 2, Ember, or whatever you're using)
-- [Lazy load code for improved initial load time](/docs/child-applications.md#load). This is baked in and requires no configuration.
-- [Ensure that no single part of the app can break everything](/docs/application-blast-radius.md).
-
-single-spa works in Chrome, Firefox, Safari, IE11, and Edge.
+- Write code using a new framework, without rewriting your existing app
+- Lazy load code for improved initial load time.
 
 ## Architectural Overview
+single-spa takes inspiration from React component lifecycles by applying lifecycles to entire applications.
+It started out of a desire to use React + react-router instead of being forever stuck with our Angular 1 + ui-router application, 
+but now single-spa has grown to now support almost any framework coexisting with any other. Since Javascript is notorious for the short-life of its
+many frameworks, we decided to make it easy to use whichever frameworks you want.
+
 Apps built with single-spa are made up of the following pieces:
 
-1. An index.html file
-1. A root application, in which child applications are registered. In order to register a child application, two things must be provided:
-  1. The url to the application
-  1. A function that determines if the application is active
-1. Many child applications, each of which is like an SPA itself. Child applications respond to url routing events and must know how to bootstrap, mount, and unmount themselves from the DOM.
+1. An html file
```

Comment:
Yeah I think I see your point here. The more I think about it, the more it seems redundant to say that you need an html file, since all web apps must have an html file by definition. I think I'll just remove this entire bullet point, since it's not helping.